picard/ui/ui_infodialog.py

```diff
@@ -16,7 +16,6 @@ def setupUi(self, InfoDialog):
         self.info_tab = QtWidgets.QWidget()
         self.info_tab.setObjectName("info_tab")
         self.vboxlayout = QtWidgets.QVBoxLayout(self.info_tab)
-        self.vboxlayout.setContentsMargins(0, 0, 0, 0)
```

Comment:
Hm, calling `build_ui` on master doesn't remove it, but calling it on this branch doesn't add it back in again, so it's not a big problem.